tests/tests_files_elasticsearch.yml

```diff
@@ -297,44 +278,81 @@
-          include_role:
-            name: linux-system-roles.logging
+    - name: "TEST CASE 2; Elasticsearch config -
+      local certs are not copied to the target host as tls is false"
+      vars:
+        logging_outputs:
+          - name: elasticsearch_output
+            type: elasticsearch
+            server_host: logging-es
+            server_port: 9200
+            index_prefix: project.
+            input_type: ovirt
+            retryfailures: false
+            tls: false
+            ca_cert_src: "{{ __test_ca_cert }}"
+            cert_src: "{{ __test_cert }}"
+            private_key_src: "{{ __test_key }}"
+        logging_inputs:
+          - name: files_input
+            type: files
+            input_log_path: "{{ __test_inputfiles_dir }}/*.log"
+        logging_flows:
+          - name: flow_0
+            inputs: [files_input]
+            outputs: [elasticsearch_output]
+      include_role:
+        name: linux-system-roles.logging
 
-        - name: unreachable task
-          fail:
-            msg: UNREACH
+    # notify restart rsyslogd is executed at the end of this test task.
+    # thus we have to force to invoke handlers
+    - name: "Force all notified handlers to run at this point,
+      not waiting for normal sync points"
+      meta: flush_handlers
 
-      rescue:
-        - debug:
-            msg: "Caught an expected error -
-              {{ ansible_failed_result.results }}"
-        - name: assert...
-          assert:
-            that: ansible_failed_result.results.0.msg is match(__expected_err1)
+    - name: Ensure config file size and counts
+      vars:
+        __conf_count: 9
+        __conf_size: less
+        __conf_files:
+          - "{{ __test_inputfiles_conf }}"
+          - "{{ __test_outputfiles_conf }}"
+        __check_systemctl_status: true
+      include_tasks: tasks/check_daemon_config_files.yml
+
+    - name: Ensure installed packages
+      vars:
+        __expected:
+          - rsyslog
+          - ca-certificates
+          - rsyslog-elasticsearch
+      include_tasks: tasks/check_packages.yml
+
+    - name: Check if the output files config exists
+      stat:
+        path: "{{ __test_outputfiles_conf }}"
+
+    - name: Check if the copied key/certs files do not exist
+      stat:
+        path: "{{ item }}"
+      register: __result
+      failed_when: __result.stat.exists
+      loop:
+        - "{{ __test_ca_cert_target }}"
+        - "{{ __test_cert_target }}"
+        - "{{ __test_key_target }}"
+
+    - name: Check key/certs in {{ __test_outputfiles_conf }}
+      command: >-
+        /bin/grep 'tls.{{ item.key }}="{{ item.value }}"'
+        {{ __test_outputfiles_conf }}
+      with_dict:
+        - cacert: "{{ __test_ca_cert_target }}"
+        - mycert: "{{ __test_cert_target }}"
+        - myprivkey: "{{ __test_key_target }}"
+      register: __result
+      changed_when: false
+      failed_when: __result.rc != 1
```

Comment:
do you use this `__result` anywhere else?  If not, then the default failed condition here is if the `grep` in the `command` returns non-zero, so I think you can get rid of the `register` and the `failed_when` here.